control/control.leanos.xml

```diff
@@ -166,6 +166,13 @@ textdomain="control"
             <version>15.3</version>
             <register_target>sle-15-$arch</register_target>
           </base_product>
+          <base_product>
+            <display_name>SUSE Linux Enterprise Real Time 15 SP3</display_name>
+            <name>SLE_RT</name>
+            <version>15.3</version>
+            <register_target>sle-15-$arch</register_target>
+            <archs>x86_64</archs>
```

Comment:
So the RT is available only for `x86_64`? (Just double checking...)

Reply:
yes, we currently only support x86_64 platform.